.github/workflows/gradle.yml

```diff
@@ -27,4 +27,4 @@ jobs:
         java-version: ${{ matrix.java }}
         cache: 'gradle'
     - name: Build with Gradle
-      run: ./gradlew clean build -x test -x checkstyleTest
+      run: ./gradlew clean build -x unitTest -x checkstyleTest
```

Comment:
why do we need this change?

Reply:
Because... `test` found zero tests. I haven't dug deeply in this gradle's filtering test logic, but just kicked `test` and reported zero tests found. So tests under `io.appium.java_client.internal` (should run as unit test) haven't kicked via this test command.